src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -16,7 +16,7 @@ CancellationToken token
 		decl = decl.WithAttributeLists( ReplaceGenerateSyncAttribute( decl.AttributeLists ) )
 			.WithModifiers( RemoveAsyncModifier( decl.Modifiers ) )
 			.WithIdentifier( RemoveAsyncSuffix( decl.Identifier ) )
-			.WithReturnType( TransformType( decl.ReturnType, optional: false ) )
+			.WithReturnType( TransformType( decl.ReturnType, isNotReturnType: false ) )

```

Comment:
nit: it might be easier to read if you remove the negation, like `isReturnType` instead. Just consider it heh.

Reply:
Avoids stuff like `!isNotReturnType` and cases where you have an if/else can just be swapped easily etc.

Reply:
I would follow this suggestion before merging.

Reply:
I was going to do this in the next PR (https://github.com/Brightspace/D2L.CodeStyle/pull/896), is that alright since I already made the changes there? Whoops

Reply:
I think it should be easy enough to bring the changes forward and resolve the differences in your follow-up. That's at least partly what git is for :smile:.